feature-group-definitions/file-and-directory-entries.dist.yml

```diff
@@ -0,0 +1,52 @@
+# Generated from: file-and-directory-entries.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: File and directory entries
+description: File and directory entries represent a virtual file system for working with files and directories dragged and dropped into the browser.
+spec: https://wicg.github.io/entries-api/
+status:
+  baseline: low
+  baseline_low_date: 2023-03-27
+  support:
+    chrome: "102"
```

Comment:
I think this feature should match these compat tables:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/webkitEntries#browser_compatibility
https://developer.mozilla.org/en-US/docs/Web/API/DataTransferItem/webkitGetAsEntry#browser_compatibility

And the differences are small enough that I think we can accept Chrome 22 and Edge 79 as the versions instead of the earliest versions.

Reply:
I don't think this matters much anymore. Firefox for Android doesn't support the entry points (I double checked this), so the status is now false. 😞

Reply:
I think our browser versions should be correct so that we don't have to reevaluate everything in order for caniuse to be able to import features. Leaving TODOs would be fine though.

Reply:
@foolip I removed an outlier, `createSyncAccessHandle` with e646c54. That pushes us back to Chrome 86, which is where a bunch of these compat features appear. What do you think?

Reply:
@ddbeck let's go with that for now. If we had https://github.com/web-platform-dx/web-features/issues/852 or something like that it would be easier to review and see at a glance where the most reasonable cutoff point is. I'll comment on that issue that we should revisit this feature.